api/4.2.0/docs/classes/core_clientinfo.clientinfo.html

```diff
@@ -129,7 +129,7 @@ <h3>labels</h3>
 					<div class="tsd-signature tsd-kind-icon">labels<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">Set</span><span class="tsd-signature-symbol">&lt;</span><span class="tsd-signature-type">string</span><span class="tsd-signature-symbol">&gt;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in core/ClientInfo.ts:48</li>
+							<li>Defined in <a href="https://github.com/hazelcast/hazelcast-nodejs-client/blob/fe23ac5a/src/core/ClientInfo.ts#L48">core/ClientInfo.ts:48</a></li>
```

Comment:
These are added in later version of typedoc and adds a link to source:

![image](https://user-images.githubusercontent.com/32355782/121906707-0cb34200-cd34-11eb-9e12-dd33780dd1f9.png)